julia_src/http.jl

```diff
@@ -10,8 +10,7 @@ function job(req::HTTP.Request)
     m = xpress_model(timeout, tol)
     @info "Starting REopt with timeout of $(timeout) seconds..."
     results = reopt(m, d)
-	# fix our memory leak? https://github.com/jump-dev/CPLEX.jl/issues/185
-	m = nothing
+	finalize(backend(m))
```

Comment:
Did this help?

Reply:
It appears to: I have run a couple 100 scenarios on our develop servers without seeing memory growth. I haven't had time to chart it yet but I will.